### PR TITLE
[CDAP-13582] Add link to prediction dataset for each model under an experiment

### PR DESCRIPTION
**Pull request by ajainarayanan**
JIRA: https://issues.cask.co/browse/CDAP-13582
Build: https://builds.cask.co/browse/CDAP-UDUT7-2/log

### CODE REVIEW

File: cdap-ui/app/cdap/components/Experiments/DetailedView/PredictionDatasetExploreModal/index.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        fetchTables(getCurrentNamespace())
+      );
+    }
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (this.state.predictionDataset !== nextProps.predictionDataset) {
+      this.setState({
+        predictionDataset: nextProps.predictionDataset
+      });
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/Experiments/DetailedView/PredictionDatasetExploreModal/index.js]: you don't need to fetchDatasetDetails in here?

**Review by elfenheart — APPROVED** [review]
👍 LGTM

**Comment by bdmogal** [comment]
Not exactly related to this PR, but can you fix the spacing between the label and value here:
image

**Review by bdmogal — APPROVED** [review]
Please change the tooltip on the Explore Predictions button to: "Explore predictions generated by this model". Rest LGTM

**Pull request merged**